src/shared/components/oncoprint/DataUtils.ts

```diff
@@ -352,21 +352,27 @@ export function fillClinicalTrackDatum(
         fillNoDataValue(trackDatum, attribute);
     } else {
         if (attribute.datatype.toLowerCase() === "number") {
-            let numValCount = 0;
-            let numValSum = 0;
+            let values = [];
             for (const x of data) {
                 const newVal = parseFloat((x as ClinicalData).value+"");
                 if (!isNaN(newVal)) {
-                    numValCount += 1;
-                    numValSum += newVal;
+                    values.push(newVal);
                 }
             }
-            if (numValCount === 0) {
+            if (values.length === 0) {
                 fillNoDataValue(trackDatum, attribute);
             } else {
-                // average
-                trackDatum.attr_val = numValSum / numValCount;
-                trackDatum.attr_val_counts[trackDatum.attr_val] = 1;
+                switch (attribute.clinicalAttributeId) {
+                    case "MUTATION_COUNT":
+                        // max
+                        trackDatum.attr_val = values.reduce((max, nextVal)=>Math.max(max, nextVal), Number.NEGATIVE_INFINITY);
```

Comment:
is this different from just Math.max(values) ?

Reply:
was not aware of this